MySensors.h

```diff
@@ -153,7 +153,7 @@ MY_DEFAULT_RX_LED_PIN in your sketch instead to enable LEDs
 
 // GATEWAY - COMMON FUNCTIONS
 // We support MQTT Client using W5100, ESP8266 and Linux
```

Comment:
Perhaps update this comment to also include GSM as communications medium now

Reply:
Done!